packages/typescript-estree/src/parser.ts

```diff
@@ -496,6 +496,12 @@ function parseWithNodeMaps<T extends TSESTreeOptions = TSESTreeOptions>(
   return parseWithNodeMapsInternal(code, options, true);
 }
 
+let parseAndGenerateServicesCalls: { [fileName: string]: number } = {};
+// Privately exported utility intended for use in typescript-eslint unit tests only
+function ɵclearParseAndGenerateServicesCalls(): void {
```

Comment:
why does this have `ɵ`? could we just export it without it?
We could just explicitly not export this from our [`index.ts`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/typescript-estree/src/index.ts)

Reply:
Yeah I actually did that at first and then applied this `ɵ` (popularised by the Angular framework codebase) just to avoid the refactor away from `export *` in index...

Not sure why I felt that was quite such a big deal and will update 👍